akka-stream-tests/src/test/scala/akka/stream/impl/fusing/ActorGraphInterpreterSpec.scala

```diff
@@ -13,12 +13,13 @@ import akka.stream.testkit.Utils._
 import akka.stream.testkit.{ StreamSpec, TestPublisher, TestSubscriber }
 import akka.testkit.EventFilter
 
-import scala.concurrent.Await
+import scala.concurrent.{ Await, ExecutionContext, Future }
 import scala.concurrent.duration._
 import org.reactivestreams.{ Publisher, Subscriber, Subscription }
 
 class ActorGraphInterpreterSpec extends StreamSpec {
   implicit val materializer = ActorMaterializer()
+  implicit def ec: ExecutionContext = materializer.executionContext
```

Comment:
But what did you change with the `ActorGraphInterpreterSpec` that makes it need them, as far as I can see you only reformatted two test cases?